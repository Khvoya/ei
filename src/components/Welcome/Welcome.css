 .Welcome {
     margin-top: 113px;
     display: grid;
     background-color: #000000;
     background-size: cover;
     background-position: center;
     background-repeat: no-repeat;
     height: 80vw;
     min-height: 70vh;
 }
 .Welcome .container {
     align-self: end;
 }
 .Welcome__logo-container {
     background-color: black;
     margin: -2px 0;
 }
 .Welcome__logo-container-wrap {
     height: 15vw;
     display: flex;
     flex-direction: row;
     justify-content: space-around;
 }
 .Welcome__logo-container a {
     position: relative;
     width: 6vw;
     height: 6vw;
     border-radius: 50%;
     border: 3px white;
 }
 .Welcome__logo {
     top: 0;
     left: 0;
     position: absolute;
     width: 6vw;
     height: 6vw;
     border-radius: 50%;
 }
 .Welcome__logo:hover {
     border: 7vw white;
     top: -5px;
     left:-5px;
     width: 7vw;
     height: 7vw;
     box-shadow: 0 0 20px 5px white , 0 0 20px 5px white inset;
     transition: all .4s;
 }

@media screen and (max-width: 600px) {
    .Welcome {
        background-size: 150%;
        background-position: top;
    }
    .Welcome__logo {
        width: 10vw;
        height: 10vw;
        top: -6px;
        left: -6px;
    }
}
