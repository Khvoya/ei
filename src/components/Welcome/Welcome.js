import React, { Component } from "react";
import "./Welcome.css";
import instagram from "../../../src/media/icons/instagram-logo.svg";
import facebook from "../../../src/media/icons/facebook-logo-button.svg";
import vk from "../../../src/media/icons/vk-social-logotype.svg";
import youtube from "../../../src/media/icons/youtube-logotype.svg";
import twitter from "../../../src/media/icons/twitter-logo-button.svg";
import bandcamp from "../../../src/media/icons/bandcamp-logo.svg";
import { Container, Row } from "react-bootstrap";

class Welcome extends Component {
  render() {
    return (
      <div>
        <div id="welcome" className="Welcome">
        </div>
        <div className="Welcome__logo-container">
          <Container >
            <Row className="Welcome__logo-container-wrap align-content-center justify-content-around ">
              <a href="https://www.instagram.com/electricindians/" target="blank">
                <img className="Welcome__logo" src={instagram} />
              </a>
              <a href="https://www.facebook.com/electricindians/" target="blank">
                <img className="Welcome__logo" src={facebook} />
              </a>
              <a href="https://vk.com/electricindians" target="blank">
                <img className="Welcome__logo" src={vk} />
              </a>
              <a
                href="https://www.youtube.com/channel/UCab_uVuxFhrwyARbotaz4Tg"
                target="blank"
              >
                <img className="Welcome__logo" src={youtube} />
              </a>
              <a href="https://electricindians.bandcamp.com/" target="blank">
                <img className="Welcome__logo" src={bandcamp} />
              </a>
              <a href="https://twitter.com/Electricindian2" target="blank">
                <img className="Welcome__logo" src={twitter} />
              </a>
            </Row>
          </Container>
        </div>
      </div>
    );
  }
}

export default Welcome;
