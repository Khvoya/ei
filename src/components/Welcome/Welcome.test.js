import React from "react";
import { shallow } from "enzyme";
import Welcome from "./Welcome";

describe("Welcome", () => {
  const props = {
    isLoaded: true,
    url:
      "https://firebasestorage.googleapis.com/v0/b/electrâ€¦=media&token=3336a0d6-a7f3-476d-8ff8-b09575f89744"
  };
  it("should render correctly", () => {
    const output = shallow(<Welcome {...props}/>);
    expect(output).toMatchSnapshot();
  });
});
