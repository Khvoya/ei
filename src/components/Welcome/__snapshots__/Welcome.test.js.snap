// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome should render correctly 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Connect(Welcome)
    isLoaded={true}
    url="https://firebasestorage.googleapis.com/v0/b/electrâ€¦=media&token=3336a0d6-a7f3-476d-8ff8-b09575f89744"
  />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "checkPropTypes": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "function",
    "props": Object {
      "children": [Function],
    },
    "ref": null,
    "rendered": [Function],
    "type": Object {
      "$$typeof": Symbol(react.context),
      "_calculateChangedBits": null,
      "_context": Object {
        "$$typeof": Symbol(react.context),
        "Consumer": [Circular],
        "Provider": Object {
          "$$typeof": Symbol(react.provider),
          "_context": [Circular],
        },
        "_calculateChangedBits": null,
        "_currentRenderer": null,
        "_currentRenderer2": null,
        "_currentValue": null,
        "_currentValue2": null,
        "_threadCount": 0,
      },
    },
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "function",
      "props": Object {
        "children": [Function],
      },
      "ref": null,
      "rendered": [Function],
      "type": Object {
        "$$typeof": Symbol(react.context),
        "_calculateChangedBits": null,
        "_context": Object {
          "$$typeof": Symbol(react.context),
          "Consumer": [Circular],
          "Provider": Object {
            "$$typeof": Symbol(react.provider),
            "_context": [Circular],
          },
          "_calculateChangedBits": null,
          "_currentRenderer": null,
          "_currentRenderer2": null,
          "_currentValue": null,
          "_currentValue2": null,
          "_threadCount": 0,
        },
      },
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "legacyContextMode": "parent",
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getChildContext": Object {
            "calledByRenderer": false,
          },
          "getDerivedStateFromProps": Object {
            "hasShouldComponentUpdateBug": false,
          },
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
  Symbol(enzyme.__childContext__): null,
}
`;
